### PR TITLE
feat(sink-worker): enable kafka client logging

### PR DESCRIPTION
**Pull request by chrisgacsal**
## Overview

Allow setting debug context for Kafka client configuration in order to get more detailed logging emitted by the `librdkafka`.

See: https://github.com/confluentinc/librdkafka/blob/master/INTRODUCTION.md#debug-contexts

## Notes for reviewer

This change allows us to have a better insights how Kafka clients interact with brokers and helps with spotting potentional issues which are not (properly) surfaced.

### CODE REVIEW

File: config/ingest.go [hunk 1]
```diff
@@ -64,6 +64,10 @@ type KafkaConfiguration struct {
 	// in case of large clusters where changes are more frequent.
 	// This value must not be set to value lower than 10s.
 	TopicMetadataRefreshInterval pkgkafka.TimeDurationMilliSeconds
+
+	// Enable contexts for extensive debugging of librdkafka.
+	// See: https://github.com/confluentinc/librdkafka/blob/master/INTRODUCTION.md#debug-contexts
+	DebugContexts []string
```
Comment (sagikazarmark) [inline config/ingest.go:70]: Is this an exhaustive list? Do we validate invalid values?

File: config/ingest.go [hunk 2]
```diff
@@ -134,6 +138,10 @@ func (c KafkaConfiguration) CreateKafkaConfig() kafka.ConfigMap {
 		config["metadata.max.age.ms"] = 3 * c.TopicMetadataRefreshInterval
 	}
 
+	if c.DebugContexts != nil {
+		config["debug"] = strings.Join(c.DebugContexts, ",")
+	}
```
Comment (sagikazarmark) [inline config/ingest.go:143]: Empty list may result in an empty string. Might be better to check for length?

**Review by sagikazarmark — DISMISSED** [review]

File: config/ingest.go (hunk 2 above)
Comment (chrisgacsal) [reply config/ingest.go:143]: Valid. Will fix it promptly.

File: config/ingest.go (hunk 1 above)
Comment (chrisgacsal) [reply config/ingest.go]: Fixed.

File: config/ingest.go (hunk 2 above)
Comment (chrisgacsal) [reply config/ingest.go]: Fixed.

**Review by sagikazarmark — APPROVED** [review]

**Pull request merged**